.github/workflows/oci.yml

```
@@ -0,0 +1,53 @@
+name: "Unit tests"
+
+on:
+  push:
+
+jobs:
+  phpunit-oci8:
+    name: "PHPUnit on OCI8"
```

Same here, something like ?
```suggestion
    name: php${{ matrix.php-versions }}-OCI8

```